README.md

```diff
@@ -56,14 +56,23 @@ to pad out the color selection, no extra configuration needed.
 
 ### Manually
 
-Put `srcery.vim` in `~/.vim/colors/` (on unix-like systems) or `%userprofile%\vimfiles\colors\` (on Windows).
+Put `srcery.vim` in `~/.vim/colors/` (on unix-like systems) or `%userprofile%\vimfiles\colors\` (on Windows). You can also install with your favourite plugin manager.
```

Comment:
This instruction is not valid anymore right? `srcery.vim` does not contain all the code for a complete color scheme.

So, my understand is that you **_must_** use the package managed to make sure you get the whole scheme?  Or am I missing something? 😅

Reply:
We might have to change the wording a bit.

It's not a single file, so that wording must be changed to say either clone the repo or copy and paste manually the entire structure. 

In terms of relying or not on a plugin manager/using vim8 packages; I think that if you set your runtime (normally by appending) to whatever folder srcery was downloaded to, it should work. Will test and report back.

Reply:
Okay @MindTooth I've made it work like I said. If you can test it as well I would appreciate it, but here are the steps I did:
1. Clone srcery-vim to a directory.
2. Set your `rtp` to also find that directory `set rtp+=/home/breno/Documents/guergeiro/srcery-vim` (this is where I cloned).
3. Load your colorscheme.

I await your report 🙂

Reply:
What I did in my Raspberry Pi, which has NO config at all:

1. `mkdir -p ~/.vim/pack/themes/opt`
2. `cd .vim/pack/themes/opt/`
3. `git clone https://github.com/srcery-colors/srcery-vim.git`
4. Inside a Vim window (open a YAML file):
    1. `:packadd srcery-vim`
    2. `:colorscheme srcery`
5. Victory! 🥳 

However, which I guess is because of the variable checks.  Is that Vim broke if I tried to run `:colorscheme` before `:packadd`, and then `:colorscheme` in an attempt to fix it.  Meaning it was broken until I restarted Vim.  Something to be vary of if people perhaps issues with their config and experience the variable issue.

Reply:
Hmm. 🤔  If I have no `.vimrc`, I'm able  to use the `:packadd` and `:colorscheme` without any issues and colors change instantaneous in the opened YAML file.  If, however, I have an empty or commented `.vimrc` file, this breaks stuff.

Reply:
My bad.  I needed the following to make it work:

```vimrc
" ~/.vim/vimrc
source $VIMRUNTIME/defaults.vim

packadd srcery-vim
colorscheme srcery
```

Ref. [`:help vimrc-intro`](https://vimhelp.org/usr_05.txt.html#05.1).

Reply:
Yeah, I know, but that uses native package manager, I'm trying to check out if people who don't have access to packages (or any plugin manager) can simply change the `rtp` to point to the srcery folder and have access to everything.

Reply:
I can verify that by appending the path to `rtp`, I'm able to run `:colorscheme srcery`.

Reply:
So what do you think? In the changes I proposed I say to users that they should set their rtp correctly. Should we just ditch that or keep it?

Reply:
That's only valid if you set the `rtp` to a clone of the project, not the `srcery.vim` itself.  Hence, the current text is invalid as you now require more than just one file.

I might be mistaken though.  But, going forward, we either instruct Vim users to either clone the project, set their `rtp` to include the clone, or use the native Vim package manager (`~/.vim/pack`).  That's my understand of the situation.  Please correct me. 😄

Reply:
I think we are on the same line. May I do changes then?

Reply:
Go ahead. :)  Let's be explicit about what people need to do correctly.

Reply:
Solved in 398a4e264119b8c8c09e6926733b100b054f15a0